src/main/java/com/firebolt/jdbc/metadata/FireboltDatabaseMetadata.java

```diff
@@ -140,7 +140,7 @@ public ResultSet getColumns(String catalog, String schemaPattern, String tableNa
 						String.valueOf(columnInfo.getPrecision()), null, // buffer length (not used, see Javadoc)
 						String.valueOf(columnInfo.getScale()), String.valueOf(COMMON_RADIX), // radix
 						columnDescription.getInt("is_nullable") == 1 ? columnNullable : columnNoNulls,
-						columnInfo.getColumnName(), // description of the column
+						null, // description of the column
```

Comment:
What's this change about?

Reply:
I made a mistake (I set the description of the column as the column name), so now instead I'm setting it to null